src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -273,7 +271,8 @@ def test_WINNF_FT_S_FAD_2(self, config_filename):
                                    'url': sas_test_harness.getBaseUrl()})
 
     # Injecting the PAL Records of the PPA into SAS UUT.
-    self._sas_admin.InjectPalDatabaseRecord(config['palRecords'])
+    for pal_record in config['palRecords']:
+      self._sas_admin.InjectPalDatabaseRecord(pal_record[0])
```

Comment:
Once you correct the config error, this should be `self._sas_admin.InjectPalDatabaseRecord(pal_record)`.

Reply:
changed accordingly